app/models/runtime/organization.rb

```diff
@@ -5,6 +5,19 @@ class Organization < Sequel::Model
 
     one_to_many :spaces
 
+    many_to_one :isolation_segment_model,
+      primary_key: :guid,
+      key: :default_isolation_segment_guid
+
+    many_to_many :isolation_segment_models,
+      left_key: :organization_guid,
+      left_primary_key: :guid,
+      right_key: :isolation_segment_guid,
+      right_primary_key: :guid,
+      join_table: :organizations_isolation_segments,
+      before_add: proc { raise CloudController::Errors::ApiError.new_from_details('OperationNotImplemented', 'add_isolation_segment', 'Organization') },
```

Comment:
501 I think should be correct here. Right now, there are no endpoints to add an isolation segment to an organization. Or to word it another way, you should not be able to do `organization.add_isolation_segment(isolation_segment.guid)`. This is a safeguard for developers, as it does not use all of the logic when adding an isolation segment (setting the default, etc).